external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/SerializableCredentialsProvider.scala

```diff
@@ -83,3 +84,146 @@ private[kinesis] final case class STSCredentialsProvider(
     }
   }
 }
+
+@InterfaceStability.Stable
```

Comment:
I wouldn't call these `Stable` just yet :) Let's call it evolving for one release cycle